Jenkinsfile_k8s

```diff
@@ -260,5 +265,32 @@ pipeline {
         }
       }
     }
+    stage('Build Docker Manifest') {
+      when {
+        expression {
+          return env.TAG_NAME != null
+        }
+      }
+      environment {
+        agent_type = 'ubuntu-20.04'
+      }
+      steps {
+        script {
+          infra.withDockerPushCredentials {
+            sh 'docker manifest create \
+                jenkinsciinfra/jenkins-agent-${agent_type}:latest \
+                --amend jenkinsciinfra/jenkins-agent-${agent_type}:arm64 \
+                --amend jenkinsciinfra/jenkins-agent-${agent_type}:amd64'
+            sh 'docker manifest push jenkinsciinfra/jenkins-agent-${agent_type}:latest'
+
+            sh 'docker manifest create \
+                jenkinsciinfra/jenkins-agent-${agent_type}:${TAG_NAME} \
+                --amend jenkinsciinfra/jenkins-agent-${agent_type}:arm64 \
+                --amend jenkinsciinfra/jenkins-agent-${agent_type}:amd64'
+            sh 'docker manifest push jenkinsciinfra/jenkins-agent-${agent_type}:${TAG_NAME}'
```

Comment:
```suggestion
            sh 'docker manifest push jenkinsciinfra/jenkins-agent-"${agent_type}":"${TAG_NAME}"'
```
- nitpick for the quotes around the "agent_type" variable
- Not nitpick, but not blocker either: it's safer to "protect" variables such as `TAG_NAME` that could contains random characters (depends on the value the user set for the tag)